python-main.js

```diff
@@ -481,6 +481,18 @@ function web_editor(config) {
     // the user sees their code or, in the case of a new program, uses some
     // sane defaults.
     function setupEditor(message, migration) {
+    
+     // Set the current language to bold in the dropdown menu
+        $(document).ready(function(){
+            $("ul.tree > li").each(function(){
+                $(this).click(function(){
+                    $("ul.tree > li").css('font-weight','normal');
+                    $(this).css('font-weight','bold');
+                });                   
+            });
+            $('ul.tree > li #en').trigger('click');
```

Comment:
Triggering a click here will always set the language to english, overwriting the URL language parameter, which can be triggered  like this: `http://localhost:8000/editor.html?l=es`

The URL language is set from this line (`qs` contains the URL parameters): https://github.com/bbcmicrobit/PythonEditor/blob/9ea5aba1ba4e34ff88c8bb3cfb50db9bcaf722d2/python-main.js#L1618